src/hardware/configuration.rs

```diff
@@ -42,9 +60,8 @@ impl NetStorage {
             )],
             neighbor_cache: [None; 8],
             routes_cache: [None; 8],
-            sockets: [None, None],
-            tx_storage: [0; 4096],
-            rx_storage: [0; 4096],
+            sockets: [None, None, None, None, None],
```

Comment:
In this case, `None` does not impl `Copy`, because the underlying `SocketSetItem` that the option wraps does not implement `Copy`. As such, we can't use `None; 5`. I think if we upped to a minimum supported rust version (MSRV) of 1.51, we could do this, but with older rust releases, it's not possible.

Reply:
Ack!